02-electric/electric-hero/electric-hero.css

```diff
@@ -29,19 +30,41 @@
   background-color: var(--level2-background-color);
 }
 
-.electric-hero-img-container{
+@media (min-width:500px){
+  .electric-hero-subcontainer{
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-areas:
+      "electric-hero-image-container electric-hero-copy-container";
+    column-gap: var(--subcontainer-column-gap-desktop);
+  }
+}
+
+.electric-hero-image-container{
   grid-area:electric-hero-image-container ;
   display: grid;
   justify-content: center;
   align-content: center;
   background-color: var(--level3-background-color);
 }
 
+@media (min-width: 500px) {
+  .electric-hero-image-container{
+    display: grid;
+    align-content: center;
+  }
+}
+
 .electric-hero-image{
   display: block;
   width: 100%;
 }
 
+@media (min-width:500px){
+  .electric-hero-image{
+
+  }
+}
+
 .electric-hero-copy-container {
   grid-area:electric-hero-copy-container ;
```

Comment:
space between property and value
grid-area:_value

Reply:
done